packages/create-video/src/create-directory.ts

```diff
@@ -0,0 +1,79 @@
+import fs from 'fs-extra';
+import * as path from 'path';
+import {Log} from './log';
+
+const TOLERABLE_FILES = [
+	// System
+	'.DS_Store',
+	'Thumbs.db',
+	// Git
+	'.git',
+	'.gitattributes',
+	'.gitignore',
+	// Project
+	'.npmignore',
+	'.travis.yml',
+	'LICENSE',
+	'docs',
+	'.idea',
+	// Package manager
+	'npm-debug.log',
+	'yarn-debug.log',
+	'yarn-error.log',
+];
+
+export function validateName(name?: string): string | true {
+	if (typeof name !== 'string' || name === '') {
+		return 'The project name can not be empty.';
+	}
+	if (!/^[a-z0-9@.\-_]+$/i.test(name)) {
+		return 'The project name can only contain URL-friendly characters (alphanumeric and @ . -  _)';
+	}
+	return true;
+}
+
+export function getConflictsForDirectory(
```

Comment:
So if you enter a folder name that already exists it will trigger this function? Should we just simplify it and exit with an error if that directory already exists instead?

Reply:
yeah I think simplification is the way to go.

Reply:
It should mean this file can be removed now right?